SPECS/btrfs-progs/btrfs-progs.spec

```diff
@@ -1,6 +1,6 @@
 Name:       btrfs-progs
-Version:    5.16
-Release:    102%{?dist}
+Version:	6.8
```

Comment:
Idk why it looks weird here, in my VS code it looks perfect:
<img width="271" alt="image" src="https://github.com/microsoft/azurelinux/assets/38226164/bd59ad2e-8e7c-4585-a696-0e802896f5cb">

Reply:
Try turning on "Unicode Highlight: Invisible Characters" setting in VsCode. It looks like you've got tab characters that should be replaced.

Reply:
Fixed!
Thank you for a hint